src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java

```diff
@@ -90,7 +90,6 @@ public Object apply(WebDriver driver) {
         }
       });
     } catch (TimeoutException e) {
```

Comment:
To resolve this Sonar issue you can now re-use @drets's new method: https://github.com/Wikia/selenium-tests/pull/797/files#diff-09e6725174ba92baabf453454b8a1d4aR117 :)